packages/wallets/src/signers/email/evm-email-signer.ts

```
@@ -0,0 +1,85 @@
+import type { EmailInternalSignerConfig } from "../types";
+import { EmailSignerApiClient } from "./email-signer-api-client";
+import { EmailSigner, DEFAULT_EVENT_OPTIONS } from "./email-signer";
+import type { Crossmint } from "@crossmint/common-sdk-base";
+import { Address, PublicKey } from "ox";
+
+export class EvmEmailSigner extends EmailSigner {
+    constructor(config: EmailInternalSignerConfig) {
+        super(config);
+    }
+
+    locator() {
+        return `evm-keypair:${this.config.signerAddress}`;
+    }
+
+    async signMessage() {
+        return await Promise.reject(new Error("signMessage method not implemented for email signer"));
+    }
+
+    async signTransaction(transaction: string): Promise<{ signature: string }> {
+        await this.handleAuthRequired();
+
+        const hexString = transaction.startsWith("0x") ? transaction.slice(2) : transaction;
+
+        const res = await this.config._handshakeParent?.sendAction({
+            event: "request:sign",
+            responseEvent: "response:sign",
+            data: {
+                authData: {
+                    jwt: this.config.crossmint.experimental_customAuth?.jwt ?? "",
```

If the jwt field is not defined we should raise an error! Doing validation here will always be easier than propagating errors in the iframe handler